tests/python/pants_test/pants_run_integration_test.py

```diff
@@ -163,6 +163,24 @@ class PantsRunIntegrationTest(unittest.TestCase):
 
   PANTS_SCRIPT_NAME = 'pants'
 
+  @classmethod
+  def should_configure_pantsd(cls):
```

Comment:
The meaning of this name is a bit ambiguous to me. Does this function returning `True` mean yes use the global env var, or that the tests "should configure pantsd" for themselves?

Maybe rename to something like `read_pantsd_env_var` or `check_for_pantsd_env_var`.